sdv/constraints/tabular.py

```diff
@@ -1109,7 +1112,7 @@ def _validate_inputs(cls, **kwargs):
         errors = []
         try:
             super()._validate_inputs(**kwargs)
-        except MultipleConstraintsErrors as mce:
```

Comment:
minor: can we change this to `as agg_error`

Reply:
Good one!